app/views/govuk_publishing_components/components/_contents_list.html.erb

```diff
@@ -12,40 +12,44 @@
                               else
                                 'dashed'
                               end
+  brand ||= false
+  brand_helper = GovukPublishingComponents::AppHelpers::BrandHelper.new(brand)
+  hide_title ||= false
 %>
 <% if contents.any? %>
   <nav
     role="navigation"
-    class="gem-c-contents-list <%= 'gem-c-contents-list--no-underline' unless underline_links %>"
+    class="gem-c-contents-list <%= 'gem-c-contents-list--no-underline' unless underline_links %> <%= brand_helper.brand_class %>"
     data-module="track-click"
     <% if aria_label.present? %>
       aria-label="<%= aria_label %>"
     <% end %>
   >
-    <h2>
-      <%= t("components.contents_list.contents") %>
-    </h2>
+    <% if not hide_title %>
```

Comment:
Can we use `unless` here for consistency?

Reply:
Ah, of course. Thanks, changed.